TELCODOCS-1110: Move LVM Storage documentation to openshift-docs (#55472)
openshift/openshift-docs: modules/lvms-scaling-storage-of-single-node-open-proc.adoc

// Module included in the following assemblies:
//
// storage/logical-volume-manager-storage.adoc

:_content-type: PROCEDURE
[id="lvms-proc-scaling-storage-of-single-node-openshift-cluster_{context}"]
= Scaling up storage by adding capacity to your {sno} cluster

To scale the storage capacity of your configured worker nodes on a {sno} cluster, you can increase the capacity by adding disks.

.Prerequisites

* You have additional unused disks on each {sno} cluster to be used by logical volume manager storage.

.Procedure

> . Log in to OpenShift console of the {sno} cluster.

Reviewer: ```suggestion
. Log in to {product-title} console of the {sno} cluster.
```

. From the *Operators* -> *Installed Operators* page, click on the *LVM Storage Operator* in the `openshift-storage` namespace.
. Click on the *LVMCluster* tab to list the `LVMCluster` created on the cluster.
. Select *Edit LVMCluster* from the *Actions* drop-down menu.
. Click on the *YAML* tab.

> . Edit the `LVMCluster` YAML to add the new device path in the `deviceSelector` section:

Reviewer: ```suggestion
. Edit the `LVMCluster` CR YAML to add the new device path in the `deviceSelector` section:
```

+
[NOTE]
====
In case the `deviceSelector` field is not included during the `LVMCluster` creation, it is not possible to add the `deviceSelector` section to the CR.
You need to remove the `LVMCluster` and then create a new CR.
====

+
[source,yaml]
----

> [...]

Reviewer: ```suggestion
...
```
